### DISCUSSION TITLE
About status icons

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#6912] [Q&A] [answered]
I found that the icon of the article status can only reference svg, right? It cannot directly use the built-in icon library, right?
![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/1363f0c1-8446-48e8-b163-047b749ee44e)

such as

:root {
    --md-status--test:  :material-test-tube:
}

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @W1ndys,
the `:material-test-tube:` syntax is limited to the Markdown (.md) content as it's later parsed with the pymdown markdown extension. So you're right you have to use a url with the SVG in the CSS override for the status type.

Using https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/ you can search for an emoji and later right click on the result and open the SVG image in a new tab:
https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/material/templates/.icons/material/test-tube.svg
then right click again and open the source view:
`view-source:https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/material/templates/.icons/material/test-tube.svg`

Copy that 1 single line text into the CSS in a special url, make sure it's all on a single line:
`--md-status--test: url('data:image/svg+xml;charset=utf-8,<svg.....');`

Reference: 
- https://github.com/squidfunk/mkdocs-material/discussions/6811

**Reply by w1ndys** [reply]
Yes, thank you very much for your reply. It is worth noting that you provided me with a reference issue, which opened up new ideas for me. You are great！
